spacepy/pycdf/const.py

```diff
@@ -19,7 +19,7 @@
 # Regeneration process:
 # Edit a copy of cdf.h
 # M-x replace-string (CDFstatus) with nothing
-# h2py cdf.h
+# h2py cdf.h (probably need to use Python 2 to capture the Longs)
```

Comment:
Does this update need "h2py.py", and if so, where is this coming from now. My understanding is that it was dropped from the CPython repo. (The Python 2 comment is definitely making me question this...)